test/built-ins/Temporal/PlainDateTime/prototype/with/calendar-options.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.with
+description: Verify the result of calendar.fields() is treated correctly.
```

Comment:
It's not clear to me from the description what the test is actually testing — it seems to test that the options object is passed through to `dateFromFields` which is different from what the description says. I'd expect to do what the description says, we'd need to add another field in an overridden `fields()` method?

Reply:
Forgot to update this description after copy/pasting from another test; fixed.